manifest.yml

```diff
@@ -0,0 +1,12 @@
+applications:
+- name: sdx-transform-cora
+  buildpack: python_buildpack
+  health-check-type: process
+
+  env:
+    CF_DEPLOYMENT: True
+    SDX_SEQUENCE_URL: http://sdx-sequence.apps.devtest.onsclofo.uk/sequence
+    FTP_PATH: \\\\NP3RVWAPXX370\\SDX_preprod\\
```

Comment:
Since it's not preprod, I don't think we need this path. It probably can just be "\"